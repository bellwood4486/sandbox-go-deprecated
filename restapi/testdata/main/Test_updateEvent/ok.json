{
  "Title": "Test Title2",
  "Description": "Test Description2"
}