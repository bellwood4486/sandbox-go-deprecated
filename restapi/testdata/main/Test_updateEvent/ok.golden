{
  "ID": "1",
  "Title": "Test Title2",
  "Description": "Test Description2"
}